### PR TITLE
Adding User Identifiers API documentation

### PR DESCRIPTION
**Pull request by Kaldjo**
Added User Identifiers API documentation and specs. Identity Services team did not publish this endpoint yet to production. 

A further notice will be issued for when this PR is ready to be merged.

### CODE REVIEW

File: content/en/docs/apidocs-mxsdk/apidocs/user-identifiers-api.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+weight: 112
+---
+
+{{% alert color="warning" %}}
+User Identifiers API is available for Mendix Admins.
+{{% /alert %}}
+
+## 1 Introduction
+
+User Identifiers API allows Mendix Admins to perform operations related to user identifiers beloging to their company. Such as getting user UUID of certain emails.
```
Comment (JaapF) [inline content/en/docs/apidocs-mxsdk/apidocs/user-identifiers-api.md]: I think it's good to mention the projects-api.
The projects api needs a user's UUID.

Comment (Karuna-Mendix) [reply content/en/docs/apidocs-mxsdk/apidocs/user-identifiers-api.md]: There is no direct replacement of project API with User Identifiers API. I think, we can mention about User Identifiers API in the project API doc as it seems more relevant there.

File: content/en/docs/apidocs-mxsdk/apidocs/user-identifiers-api.md [hunk 2]
```diff
@@ -10,24 +10,24 @@ weight: 112
...
 
 ### 2.1 Generating a PAT
 
-To generate a PAT, see the [Personal Access Tokens](/community-tools/mendix-profile/user-settings/#pat) section of *Mendix Profile*.
+To generate a PAT, see the [Personal Access Tokens](/community-tools/mendix-profile/user-settings/#pat) section of *User Settings*.
 
 Select the following as **User Identifiers API** scopes:
 
-* `mx:mxid3:user-identifiers:uuid:read` – to perform `GET` operations on user's UUIDs
+`mx:mxid3:user-identifiers:uuid:read` – to perform `GET` operations on user's UUIDs
```
Comment (Karuna-Mendix) [inline content/en/docs/apidocs-mxsdk/apidocs/user-identifiers-api.md]: The `GET` operation mentioned here is not matching with what is described in the API Reference section. Please check.

Comment (Kaldjo) [reply content/en/docs/apidocs-mxsdk/apidocs/user-identifiers-api.md]: Good catch, It's indeed an HTTP POST request. The intention is not to push anything to the server but rather get the user UUIDs.
So perhaps it's best to not mention which operation is performed and replace it with something along the lines of 'to obtain user UUIDs'? Let me know what you think

Comment (Karuna-Mendix) [reply content/en/docs/apidocs-mxsdk/apidocs/user-identifiers-api.md]: Done https://github.com/mendix/docs/pull/7481/commits/f863a256f91bc81a57e3aaf83c295371e806fbe2

**Comment by Karuna-Mendix** [comment]
@Kaldjo, Review is done for this PR. Let me know when to merge it.

**Comment by Kaldjo** [comment]
@Karuna-Mendix, We've just released the API on production. Therefore this PR can be merged.

Thank you for your efforts!

**Pull request merged**